packages/core/src/domain/configuration/configuration.ts

```diff
@@ -112,7 +112,13 @@ export function validateAndBuildConfiguration(initConfiguration: InitConfigurati
   }
 
   // Set the experimental feature flags as early as possible, so we can use them in most places
-  updateExperimentalFeatures(initConfiguration.enableExperimentalFeatures)
+  if (Array.isArray(initConfiguration.enableExperimentalFeatures)) {
+    updateExperimentalFeatures(
+      initConfiguration.enableExperimentalFeatures.filter((flag): flag is ExperimentalFeature =>
+        objectHasValue(ExperimentalFeature, flag)
+      )
+    )
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏could it be interesting to warn the customer if we filtered out a provided value?

Reply:
I'm not sure. Like we discussed, in the future `enableExperimentalFeatures` should be less customer facing, so it'll be mostly us who use it. And if we were to show a warning every time a flag does not exist anymore, it will spam our fellow Datadog frontend devs until we remove the flag from consul, which will never happen exactly at the same time.

I would keep things quiet for now.